test/mailers/mailer_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
```

Comment:
to make CI happy this change is needed

```suggestion
require "test_helper"
```